modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -164,14 +164,43 @@ app
  app:list         List all available apps
 ----
 
+==== List Available Apps
+
 List all of your installed apps or optionally provide a search pattern to restrict the list of apps to those whose name matches the given regular expression. 
 The output shows whether they are enabled or disabled.
 
 [source,console,subs="attributes+"]
 ----
-{occ-command-example-prefix} app:list [<search-pattern>]
+{occ-command-example-prefix} app:list [--] [<search-pattern>]
 ----
 
+===== Arguments
+
+[width="100%",cols="20%,70%",]
+|===
+| `search-pattern` 
+| Show only those apps whose names match the given search pattern (regular expression).
+|===
+
+===== Options
+
+[width="100%",cols="20%,70%",]
+|===
+| --disabled
+| Only display disabled apps.
```

Comment:
The core PR was updated so that the help text now says:
```suggestion
| Only display disabled apps. If the app was previously enabled, the app version is also displayed.
```
@settermjd this could be updated here, then you could adjust or remove the line below.